crates/orderbook/openapi.yml

```diff
@@ -593,6 +598,31 @@ components:
       required:
         - expiration
         - amount
+    OnchainOrderData:
+      description: |
+        Provides the additional data for onchain placed orders
+      type: object
+      properties:
+        onchainUser:
```

Comment:
For the `valid_to` you called it `user_valid_to`. I don't like the name, but `user_owner` would make more sense, right? otherwise, `original_owner` seems more clear.

Reply:
User valid is a concept used only for the ethflow. Onchain orders are more generic. In order to indicate that this is available for all on-chain order placed, I think onchain_user is a good name.

Reply:
I'm fine with this, just a nitpick that everything uses snake_case and this property uses camelCase

Reply:
fine with me, although my bring in the last nitpick. I feel like technically both users are onchain, so `onchainUser` don't leave it automatically clear. `original_owner` seems slightly more clear to me.

Reply:
Yeah, it's not optimal. 
I am having second thoughts and I think it would be nice to call it as it is defined in the smart contract interface: `sender`. E.g. each onchainOrder has a sender:
https://github.com/cowprotocol/ethflowcontract/blob/main/src/interfaces/ICoWSwapOnchainOrders.sol#L40